deployment/terraform/resources/function.tf

```diff
@@ -43,7 +47,7 @@ resource "azurerm_linux_function_app" "pctasks" {
     application_insights_key = azurerm_application_insights.pctasks.instrumentation_key
 
     application_stack {
-      python_version = "3.8"
+      python_version = "3.10"
```

Comment:
We need this to use the `typing.TypeAlias` imported from the `pctasks.cosmos`. Also it's probably best to keep the versions consistent across functions and what we're using in the rest of pctasks (which I think is 3.10).